main.go

```diff
@@ -35,19 +36,27 @@ func main() {
 	flag.Usage = usage
 	flag.Parse()
 
+	if len(os.Args) > 1{
+		alias := os.Args[1]
+		if alias != "" {
+			modifyUser(alias)
+		}
+	}	
+
```

Comment:
I would like to `modifyUser()` is written in `run()`, and use early return like this.

```go
func run(alias string) int {
    if alias != "" {
        if err := modifyUser(alias); err != nil {
            fmt.Fprintf(os.Stderr, ...)
            return 1
        }
        return 0
    }

    // select, add, delete or modify
}
```

Reply:
Done :)